docs/Web-Interface.md

```diff
@@ -85,6 +85,17 @@ If the web browser and the Transmission daemon are on the machine you can use ht
 | 5                 | Downloading speed                                     |
 | 6                 | Uploading speed                                       |
 
-
+### 4. List of added torrents ###
+![image](https://user-images.githubusercontent.com/76732207/181936421-a6d36ca0-e214-4f37-9368-1404cbeb125c.png)
```

Comment:
nit: IMO it's better to have these images committed into repo.

Reply:
But i dowt know, where to put them. `docs/resources` ?

Reply:
Yup, this path is ok.
You can do it for all image files listed in this markdown in a separate PR.

Reply:
Ok, thanks!

Reply:
This is a good idea. There are a few other GitHub image links in the docs too. I'd be :+1: on those being moved to `docs/resources/` as well